install.sh

```diff
@@ -777,9 +779,17 @@ get_free_range() {
 set_user_subuids_subgids() {
 
   local min_range_size=65536
-  local current_user=$(get_user)
+  local current_user="${OPT_USER:-$(get_user)}"
   local free_range=$(get_free_range)
 
+  if [[ "$current_user" == "root" ]]; then
+    abort "You cannot assign subuids/subgids to root."
+  fi
+
+  if ! id "$current_user" &>/dev/null; then
+    abort "User '$current_user' does not exist."
+  fi
```

Comment:
This should be done as early as possible. Utilize `validate:` param of `param` function in the flag parsing section:
```shell
param ... validate:...
```